beamer/chains.py

```diff
@@ -0,0 +1,82 @@
+from dataclasses import dataclass
+from typing import Any
+
+from beamer.typing import ChainId
+
+
+@dataclass(frozen=True)
+class ChainDescriptor:
+    id: ChainId
+    ecosystem_name: str
+    name: str
+    type2: bool = False
+    bedrock: bool = False
+    testnet: bool = False
+    local: bool = False
+
+
+DESCRIPTORS = (
```

Comment:
Is there a reason that this is publc? If we don't need it, I'd rather that it is private; even more so since it's mutable.